llama_index/vector_stores/jaguar.py

```diff
@@ -216,6 +216,10 @@ def add_text(
         text = text.replace("'", "\\'")
         vcol = self._vector_index
         filecol = kwargs.get("file_column", "")
+        text_tag = kwargs.get("text_tag", "")
+
+        if text_tag != "":
```

Comment:
why not put this into a separate meta data field.
this will overwrite the raw content of text right

Reply:
That is good point. We could use metadata_fields to retrieve it from the vector, just not sure if the metadata will be sent along the main text to LLM (especially in the agent environment). Another concern is the metadata normally have size limits (like 8 bytes, 32 bytes etc) for storage and retrieval efficiency.  This tag may use a few sentences or longer to give a better context for the main text.  It would be a free-format and ensured to be tagged to the main text for LLM to use. It is meant to compliment the main text to give a better context (for example, similar document data, but for different scenarios or backgrounds).

Reply:
In llama-index, metadata to nodes is by default sent to the LLM

But in any case, this seems to be a specialized field for jaguar, so we can add it.